builder/server_builder.livecodescript

```diff
@@ -110,20 +110,12 @@ command serverBuilderRun pPlatform, pEdition
       end repeat
       
       local tServerPath, tServerZipPath
-      switch pEdition
-      case "Community"
-         put tEngineFolder & slash & "server-community" & tExeExtension into tServerPath
+      put tEngineFolder & slash & "server-" & toLower(pEdition) & tExeExtension into tServerPath
+      if pEdition is "Community" then
          put "livecode-community-server" & tOutExeExtension into tServerZipPath
-         break
-      case "Indy"
-         put tEngineFolder & slash & "server-commercial" & tExeExtension into tServerPath
-         put "livecode-server" & tOutExeExtension into tServerZipPath
-         break
-      case "Business"
-         put tEngineFolder & slash & "server-professional" & tExeExtension into tServerPath
+      else
          put "livecode-server" & tOutExeExtension into tServerZipPath
```

Comment:
@livecodeian am I missing something or does this mean the commercial server edition zips will clobber each other?

Reply:
Oh I see the file name in the archive will be the same... hmm... if there's a reason to make community different here perhaps they should all be different?

Reply:
Perhaps - though the naming differences between community / commercial was there before this patch and I usually err on the side of being conservative when making these kinds of changes.

Reply:
ok, perhaps we should discuss internally as it's either important to indicate the edition and therefore license in the file name or it isn't so it seems like they should all be the same or all different.